typed-store/src/rocks/mod.rs

```diff
@@ -451,3 +425,16 @@ pub fn open_cf<P: AsRef<Path>>(
     };
     Ok(rocksdb)
 }
+
+/// TODO: Good description of why we're doing this
```

Comment:
The doc on the default Rocks comparator being BE is a bit obscure, and you'd have to look into rust-rocksdb for how this is selected every time, but we can certainly mention in a sentence that "rocksdb stores keys in BE and has a seek operator on iterators": 
https://github.com/facebook/rocksdb/wiki/Iterator#introduction